custom/icds_reports/ucr/tests/test_expressions.py

```diff
@@ -1,36 +1,48 @@
+from copy import deepcopy
 import uuid
 from django.test import TestCase, override_settings
 import mock
 from casexml.apps.case.mock import CaseStructure, CaseFactory
 from casexml.apps.case.tests.util import delete_all_cases, delete_all_xforms
-from corehq.apps.es.fake.forms_fake import FormESFake
 from corehq.apps.userreports.expressions.factory import ExpressionFactory
 from corehq.apps.userreports.specs import EvaluationContext
 from corehq.form_processor.interfaces.dbaccessors import FormAccessors
 
+es_form_cache = []
```

Comment:
I'm not crazy about these test globals - seems like they should be attached to the test class.  Doing so sounds a bit tricky, though, and it may end  up more complicated anyways, so I'm cool with it.